spikeinterface/sorters/runsorter.py

```diff
@@ -337,6 +337,8 @@ def run_sorter_container(
             raise ValueError(f"sorter {sorter_name} not in SORTER_DOCKER_MAP. Please specify a container_image.")
 
     SorterClass = sorter_dict[sorter_name]
+    default_params = SorterClass.default_params()
+    sorter_params = {**default_params, **sorter_params}
```

Comment:
```suggestion
    # sorter_params is updating default_params to access GPU-related parameters
    sorter_params = {**default_params, **sorter_params}
```

Reply:
@chyumin I don't have permissions to push.. can you apply this comment so that internally we remember why we do it?

Then it looks good to merge